week_3/15_apr/iterators.js

```diff
@@ -0,0 +1,15 @@
+console.log([1,-3,4].filter((value, index) => {
+    if (value >= 0) {
+        return true
+    }
+
+    else {
+        return false
+    }
+}));
```

Comment:
@codal-kcolaco Please find the optimized way to handle the above  in the filter.